### ISSUE TITLE
[Bug]: Cannot access private property PowerComponents\LivewirePowerGrid\Helpers\Actions::$parameters

### ISSUE THREAD

**Issue by happy-soft-solutions** [Power-Components/livewire-powergrid#635]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

No, this error is not related to views.

### Is there an error in the console?

No

### PowerGrid

3.4

### Laravel

9.31.0

### Livewire

2.10.7

### Alpine JS

3.10.3

### Theme

Tailwind 3.x with tailwind/forms

### Describe the bug.

A row action button with a defined route comes with an error.

### To Reproduce...

Sets the action’s route on a row action button.

### Extra information

```markdown
Property 'parameters' in class 'Actions' (PowerComponents\LivewirePowerGrid\Helpers) is private.

It solves my problem when it's public.
```

**Comment by happy-soft-solutions** [comment]
Downgrading to V3.3 solves also the problem -:)

**Comment by luanfreitasdev** [comment]
Hello @happy-soft-solutions , thanks for reporting this issue. I'm already uploading the fix.
Fixed in 3.4.1
